### ISSUE TITLE
Meeting: Jen-Owain (7/3)

### ISSUE THREAD

**Issue by owainlharris** [owainlharris/phd_home#7]
**Things to discuss:**
- [  ] Break down of project, e.g. three chapters, by-papers 
1. Tool to identify DIs, multiple methods/multiple papers 
2. Understand atmospheric conditions that lead to DIs
3. Evaluation of CMIP models and their ability to represent DIs and their impacts
- [  ] RTSG budget and laptop purchase
- £10.7K total, £2K for high powered laptop/software, and £1K for HPC access (est.)
- email Hywel, also ask to transfer powers to Jen for supervision
- [  ] Data & Methods 
1. Shira's data is target data (labelled)
- Binary mask, DI or no-DI is through N contours to find suitable values to identify DIs
- We're using ERA5 reanalysis data for ML to try and predict objects identified in Shira's dataset? This includes inputs of PV, specific and relative humidity, temperature lapse rate at low levels etc, from raw reanalysis data 
- Loss function of binary cross entropy 
- Output is probability of DI, must be converted to binary classifier of Y or N
2. Atmospheric conditions leading to DIs
3. Evaluation of CMIP models, compare to ERA5 reanalysis objects identified. Similar to Priestley & Catto 2022 where you look at model agreement between CMIP models, and then agreement with ERA5?
- advantages of identifying DIs at one level as opposed to multiple levels
- data from models and applying algorithm to models and comparing outputs 

**To do:**
Data
- ERA5 data on gallah (dataset ER5) 
- contours of N for identifying DI objects
- comparison of variable fields with DI objects, are there any signatures within them of DIs?
- read netcdf plots into python, Matt, Jacob, etc. 

Review
- send Jen first draft when ready, incl. name in filename
